src/api/libcellml/enumerations.h

```diff
@@ -31,6 +31,83 @@ enum CELLML_FORMATS
      CELLML_FORMAT_XML = 0 /**< XML format for CellML files. */
 };
 
+/**
+ * @brief The CELLML_PREFIXES enum.
+ * Enum dexcribing the prefixes that are available for
+ * units.
+ */
+enum CELLML_PREFIXES
```

Comment:
Since we are in the ```libcellml``` namespace, is there really a need to have ```CELLML``` everywhere? In other words, why not simply have ```PREFIXES```, ```PREFIX_YOTTA```, ```PREFIX_ZETTA```, etc. (Same for the standard units.)